android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ItemTouchHandler.java

```diff
@@ -81,6 +82,37 @@ private boolean canSwipe(TiViewProxy holderProxy)
 		return holderProxy.getProperties().optBoolean(editProperty, defaultValue);
 	}
 
+	/**
+	 * Iterate through parent stack to obtain background drawable.
+	 *
+	 * @param parentView View to iterate up.
+	 * @return Drawable
+	 */
+	private Drawable getBackground(TiUIView parentView)
+	{
+		Drawable parentBackground = null;
+
+		while (parentView != null && parentBackground == null) {
+			parentBackground = parentView.getBackground();
+
+			if (parentBackground == null) {
+				final View nativeView = parentView.getNativeView();
+
+				if (nativeView != null) {
+					parentBackground = nativeView.getBackground();
+				}
+			}
+
+			final TiViewProxy parentProxy = parentView.getParent();
+			if (parentProxy == null) {
+				break;
+			}
+			parentView = parentProxy.getOrCreateView();
+		}
+
+		return parentBackground;
+	}
+
```

Comment:
I'm still seeing this row transparency issue if I don't set "backgroundColor" on any of the proxies (Window, ListView, ListItem), which is the default and completely uses themes.

Maybe we should look at all of the parent views and use the first non-transparent color drawable like the below? Odds are the activity's root decor view will be assigned a color drawable.
```java
private ColorDrawable getBackgroundColorDrawable(TiUIView uiView)
{
	if (uiView == null) {
		return null;
	}

	View view = uiView.getNativeView();
	while (view != null) {
		Drawable drawable = view.getBackground();
		if (drawable instanceof ColorDrawable) {
			ColorDrawable colorDrawable = (ColorDrawable) drawable;
			if (colorDrawable.getColor() != Color.TRANSPARENT) {
				return colorDrawable;
			}
		}

		ViewParent viewParent = view.getParent();
		view = (ViewParent instanceof View) ? (View) viewParent : null;
	}
	return null;
}
```

Reply:
Alternatively, we can add a fallback which fetches the activity's "windowBackground" color.
```java
Activity activity = this.recyclerViewProxy.getActivity();
if (activity != null) {
	TypedValue attribute = new TypedValue();
	if (activity.getTheme().resolveAttribute(android.R.attr.windowBackground, attribute, true)) {
		if ((attribute.type >= TypedValue.TYPE_FIRST_COLOR_INT) && (attribute.type <= TypedValue.TYPE_LAST_COLOR_INT)) {
			return new ColorDrawable(attribute.data);
		}
	}
}
```